internals/reminders_test.py

```diff
@@ -37,18 +37,39 @@ def __init__(self, status_code=200, text='{}'):
 
 
 def make_test_features():
-  feature_1 = core_models.Feature(
-      name='feature one', summary='sum', owner=['feature_owner@example.com'],
-      category=1, ot_milestone_desktop_start=100)
+  feature_1 = FeatureEntry(
+      name='feature one', summary='sum',
+      owner_emails=['feature_owner@example.com'],
+      category=1, feature_type=0)
   feature_1.put()
-  feature_2 = core_models.Feature(
+  stages = [110, 120, 130, 140, 150, 151, 160]
+  for stage_type in stages:
+    stage = Stage(feature_id=feature_1.key.integer_id(), stage_type=stage_type)
+    if stage_type == 150:
```

Comment:
Can you add a comment about this conditional?

Reply:
Similarly for the other stage loops below.

Reply:
Will do - I've done this in a couple of places and they should be explained each place (this is this ID associated with the origin trial stage type of this feature type)